### PR TITLE
Create Event “2020/10/2020-10-22-women-in-federal-it-cyber-conference-panel-discussion-4-veterans-day”

### PR DESCRIPTION
**Pull request by ayerkes**
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/cms/events/2020/10/2020-10-22-women-in-federal-it-cyber-conference-panel-discussion-3-veterans-day/event/2020/11/10/women-in-federal-it-cyber-conference-panel-discussion-4-veterans-day/

### CODE REVIEW

**Review by ayerkes — COMMENTED** [review]
updating speakers text

File: content/events/2020/10/2020-10-22-women-in-federal-it-cyber-conference-panel-discussion-3-veterans-day.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# See all authors at https://digital.gov/authors
+authors:
+  - renata-spinks
+  - donna-roy
+slug: women-in-federal-it-cyber-conference-panel-discussion-3-veterans-day
+# zoom, youtube_live, adobe_connect, google
+event_platform: zoom
+youtube_id: "null"
+---
+A networking event complementing the video to be released on Monday, November 10, Service to Service, on the perspectives from women who have transitioned from serving in the armed forces to serving our country as civilian government employees. In this networking event, participants will have the opportunity to ask questions to senior leaders who have transitioned from military to civilian service in the IT and cyber field. Come prepare to ask on transitioning, leadership and career development.
```
Comment (saracope) [inline content/events/2020/10/2020-10-22-women-in-federal-it-cyber-conference-panel-discussion-3-veterans-day.md:26]: typo fix, should be "Come prepared"

Comment (saracope) [reply content/events/2020/10/2020-10-22-women-in-federal-it-cyber-conference-panel-discussion-3-veterans-day.md:26]: also: should it be "Come prepared to ask **questions** on transitioning..." 
this reads a bit weird to me

Comment (Schultea) [reply content/events/2020/10/2020-10-22-women-in-federal-it-cyber-conference-panel-discussion-3-veterans-day.md]: I removed the sentence completely. It seemed redundant to me.

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**